src/Initializer.js

```diff
@@ -1,289 +1,405 @@
-'use strict'
```

Comment:
I think this is unintended. The new file was moved to the location of the old file and it was most likely just missing it to begin with. Good catch.